tests/containers/container_engine.pm

```diff
@@ -47,7 +48,22 @@ sub run {
     my $engine = $self->containers_factory($self->{runtime});
     test_seccomp() if ($self->{runtime} eq 'docker');
 
-    # Test the connectivity of Docker containers
+    # Check default network backend (podman only)
+    # Skip on SLES < 15-SP4 because `podman info --format {{.Host.NetworkBackend}}` is not supported there (ancient podman version)
+    if ($self->{runtime} eq 'podman' && !is_sle("<15-SP4")) {
+        my $expectedBackend = is_netavark_default ? "netavark" : "cni";
+        my $backend = script_output("podman info --format '{{.Host.NetworkBackend}}'");
+        record_info("network backend", $backend);
+        if ($backend !~ $expectedBackend) {
+            if (is_tumbleweed) {
```

Comment:
Shouldn't happen anymore, no need for this softfail

Reply:
Good! :slightly_smiling_face: